pkg/util/mount/mount_linux.go

```diff
@@ -1005,6 +1005,9 @@ func (mounter *Mounter) SafeMakeDir(subdir string, base string, perm os.FileMode
 }
 
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
+	if _, err := os.Stat(pathname); os.IsNotExist(err) {
+		return nil, nil
```

Comment:
instead of nil, you may want to return an empty array.  Not sure if all the callers will be able to handle nil return if nil error